prowler/providers/azure/services/app/app_function_identity_is_configured/app_function_identity_is_configured.metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "Provider": "azure",
+  "CheckID": "app_function_identity_is_configured",
+  "CheckTitle": "Ensure Azure function has system or user assigned managed identity configured",
```

Comment:
I think we need to check if the function has system assigned identities, not user.

Reply:
[CloudTrail](https://www.trendmicro.com/cloudoneconformity/knowledge-base/azure/Functions/azure-function-user-assigned-identity.html#) separates it in two different checks, but I unify both in one because in normal web applications we have only with identity without distinguishing between system or user (the only thing I would change in the check is inside identity to check the type attribute). At the end is so similar have System Assigned or User, the only that could be change is the metadata according different recommendations for each.